makefile

```diff
@@ -0,0 +1,10 @@
+all:
+	echo "run, build, clear"
+run:
+	./primary $(ARGS)
+build:
```

Comment:
Build needs to have deps
If not, I think that make would compile everything again even if nothing changed